js/piwik.js

```diff
@@ -4827,6 +4827,9 @@ if (typeof window.Matomo !== 'object') {
             this.getContent = function () {
                 return content;
             };
+            this.isUsingAlwaysUseSendBeacon = function () {
```

Comment:
Maybe `isAlwaysUsingSendBeacon` would be a better name.

Reply:
it's only an internal name and will be removed in the release